src/test/java/bio/terra/workspace/service/workspace/flight/CreateGoogleContextFlightTest.java

```diff
@@ -147,9 +149,10 @@ private void assertRolesExist(Project project) throws IOException {
       String fullRoleName =
           "projects/" + project.getProjectId() + "/roles/" + customRole.getRoleName();
       Role gcpRole = crl.getIamCow().projects().roles().get(fullRoleName).execute();
-      assertThat(
-          gcpRole.getIncludedPermissions(),
-          containsInAnyOrder(customRole.getIncludedPermissions().toArray()));
+
+      List<String> gcpPermissions =
+          Optional.ofNullable(gcpRole.getIncludedPermissions()).orElse(Collections.emptyList());
```

Comment:
prefer empty collections instead of null collections in objects, or else all clients have to do this kind of error-prone checking.

Reply:
Agreed, but Role is a GCP client library object, so we're stuck with this approach here